vendor/github.com/kubernetes-csi/external-snapshotter/client/v6/clientset/versioned/fake/register.go

```diff
@@ -28,8 +28,10 @@ import (
 	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
 )
 
-var scheme = runtime.NewScheme()
-var codecs = serializer.NewCodecFactory(scheme)
+var (
+	scheme = runtime.NewScheme()
+	codecs = serializer.NewCodecFactory(scheme)
+)
```

Comment:
Same here.  Files under clienset are automatically generated.  They should not be changed manually.